components/TextFields/src/private/MDCTextInputCommonFundament.m

```diff
@@ -593,7 +598,7 @@ - (void)clearButtonDidTouch {
     }
   }
 
-  self.text = nil;
+  [self.textInput clearText];
```

Comment:
Why clear the text in a "clearButtonAlpha" method ? Looks like a code smell

Reply:
This is for overwriting in the subclass.

Reply:
I'm not sure I understand the response. My concern was that it is an unexpected side effect (the field being cleared when clearing the button's alpha). Is that not the case?

Reply:
Communicated offline. The `clearText` exists in `clearButtonDidTouch`